src/NServiceBus.Core.Tests/Transports/Learning/AsyncFileTests.cs

```diff
@@ -11,7 +11,7 @@ public class AsyncFileTests
         public async Task When_file_content_larger_than_buffer_size()
         {
             var originalContent = string.Join("", Enumerable.Repeat("a#~×ψؾࢯ‽%1", 2000));
-            const string filePath = "test.txt";
```

Comment:
You're saying that TestContext.CurrentContext.TestDirectory doesn't work for vstest?